iroh-net-report/src/reportgen/probes.rs

```diff
@@ -88,6 +92,16 @@ pub(super) enum Probe {
         delay: Duration,
         node: Arc<RelayNode>,
     },
+    #[display("QUIC Address Discovery Ivp4 after {delay:?} to {node}")]
```

Comment:
What would you think of calling this `QAD IPv4 after...`?  (also - typo on IPv4)

The way these are logged i think it would be nicer to have a bit shorter and I don't mind that you need to know what QAD means here.  These logs are for us.